docs/learn/dapper-video.md

```diff
@@ -16,8 +20,21 @@ This Unity Multiplayer Networking tutorial will teach you all about the new offi
         allowfullscreen
         />
 
+This Unity Multiplayer tutorial will teach you how to implement connection approval so that clients have to send the correct password to be able to connect to the server.
```

Comment:
Perhaps add a heading, especially if there may be more entries? 

```suggestion
## Lobby Password

This Unity Multiplayer tutorial will teach you how to implement connection approval so that clients have to send the correct password to be able to connect to the server.
```